js/src/components/paid-ads/assetSpecs.js

```diff
@@ -435,6 +438,59 @@ const ASSET_TEXT_SPECS = [
 		}, sharedMax );
 	}
 
+	/**
+	 * Gets the conditional tip of the maximum number of this type of asset images.
+	 *
+	 * If the total number of the selected images is less than the shared maximum number,
+	 * the tip will include the minimum image requirements.
+	 *
+	 * @param {number} sharedMax The shared maximum number of the grouped types of image assets.
+	 * @param {Object[]} specs The image specs in the same group.
+	 * @param {AssetGroupFormValues} values The assets form values to be calculated for the total number of selected asset images.
+	 * @return {string} The tip of the maximum number of this type of asset images.
+	 */
+	function getMaxNumberTip( sharedMax, specs, values ) {
+		const totalSelected = specs.reduce(
+			( acc, spec ) => acc + values[ spec.key ].length,
+			0
+		);
+
+		if ( totalSelected === sharedMax ) {
+			return sprintf(
+				// translators: The shared maximum number of the grouped types of image assets.
+				__(
+					'The maximum number of images that can be uploaded is %d.',
+					'google-listings-and-ads'
+				),
+				sharedMax
+			);
+		}
+
+		const names = specs
+			.filter( ( spec ) => spec.min > 0 )
+			.map( ( spec ) =>
+				sprintf(
+					// translators: 1: The minimum number of this asset field. 2: Asset field name.
+					__( '%1$d %2$s', 'google-listings-and-ads' ),
+					spec.min,
+					spec.lowercaseName
+				)
+			);
+
+		return sprintf(
+			// translators: 1: The shared maximum number of the grouped types of image assets. 2: Text for the minimum number and type of each image asset.
+			__(
+				'Maximum %1$d images can be uploaded, with a minimum of %2$s image.',
+				'google-listings-and-ads'
+			),
+			sharedMax,
+			concatenateAssetStrings( names )
+		);
+	}
+
+	// Maximum 20 images can be uploaded, with a minimum of 1 landscape and 1 square image.
```

Comment:
💅 I this comment deliberately added here?

Reply:
Thanks for noticing this. That's my temporary note when developing. Removed by https://github.com/woocommerce/google-listings-and-ads/pull/1911/commits/ceeb912b3ba4e7aac10d832e550586c71b039670.